client/src/components/DisplayTable/DisplayTable.js

```diff
@@ -149,10 +152,24 @@ export class DisplayTable extends React.Component {
       .map(_.last)
       .value();
 
+    const data_csv = [ordered_column_keys].concat(
+      _.map(sorted_filtered_data, (row) =>
+        _.map(ordered_column_keys, (key) =>
+          col_configs_with_defaults[key].raw_formatter
+            ? col_configs_with_defaults[key].raw_formatter(row[key])
+            : row[key]
+        )
+      )
+    );
```

Comment:
Huh, I must have missed this before, but `ordered_column_keys` is an array of the headers, not the keys? Or are we using the plain language headers as keys?

If you set a default `raw_formatter` of `_.identity` then you could skip the ternary check.

Reply:
ohh `ordered_column_keys` is keys, as name suggests